alexandria/core/views.py

```diff
@@ -52,6 +52,11 @@ class DocumentViewSet(PermissionViewMixin, VisibilityViewMixin, views.ModelViewS
     serializer_class = serializers.DocumentSerializer
     queryset = models.Document.objects.all()
     filterset_class = DocumentFilterSet
+    search_fields = (
+        "title",
```

Comment:
Consider using [`@` prefix on the fields](https://www.django-rest-framework.org/api-guide/filtering/#searchfilter) to enable PostgreSQL FTS search mode

Reply:
I prefer having the default case-insensitive partial matches here.

With postgres' full-text search, `filename` doesn't match `filename.pdf` for example.